cpp/include/raft/core/operators.hpp

```diff
@@ -40,6 +40,14 @@ struct identity_op {
   }
 };
 
+struct no_op {
+  template <typename Type, typename... UnusedArgs>
+  constexpr RAFT_INLINE_FUNCTION void operator()(const Type& in, UnusedArgs...) const
```

Comment:
Why keep `const Type& in`? The variadic arguments should be sufficient.

Reply:
Oops. Fixed.